config/coreboot-nitropad-nv41.config

```diff
@@ -603,7 +603,7 @@ CONFIG_INTEL_GMA_HAVE_VBT=y
 CONFIG_INTEL_GMA_ADD_VBT=y
 # CONFIG_SOFTWARE_I2C is not set
 CONFIG_I2C_TRANSFER_TIMEOUT_US=500000
-# CONFIG_RESOURCE_ALLOCATION_TOP_DOWN is not set
+CONFIG_RESOURCE_ALLOCATION_TOP_DOWN=y
```

Comment:
Might be it.

Reply:
Changed back to not set under [b319959](https://github.com/linuxboot/heads/pull/1723/commits/b319959650c0086e6859a73f6765165f20be8b0f)

Reply:
Rebuilding with crossgcc CircleCI cache with 
837723b at https://app.circleci.com/pipelines/github/tlaurion/heads/2675/workflows/d226fa64-4d27-4d08-a86a-f04d243c7001